#!/usr/bin/python3
'''This function raises a name exception as a message'''


def raise_exception_msg(message=""):
    raise NameError(message)
